deploy/playbooks/roles/common/tasks/postgresql.yml

```diff
@@ -9,7 +9,7 @@
   sudo: yes
   lineinfile:
      # TODO: should not hardcode that version
-     dest: /etc/postgresql/9.3/main/pg_hba.conf
+     dest: /etc/postgresql/9.5/main/pg_hba.conf
```

Comment:
I couldn't find a generic link but if there is a better option I wouldn't mind updating it.